framework/packages/abstract-std/src/native/ica_client.rs

```diff
@@ -0,0 +1,173 @@
+use crate::objects::TruncatedChainId;
+use cosmwasm_schema::QueryResponses;
+use cosmwasm_std::CosmosMsg;
+use cw_ownable::{cw_ownable_execute, cw_ownable_query};
+
+pub use action::{IcaAction, IcaActionResponse, IcaExecute};
+pub use chain_type::{CastChainType, ChainType};
+
+pub use polytone_evm::EVM_NOTE_ID;
+pub use polytone_evm::POLYTONE_EVM_VERSION;
+
+/// This needs no info. Owner of the contract is whoever signed the InstantiateMsg.
```

Comment:
We can get rid of the addresses here and use the deterministic addresses. 

Also would be nice to have an owner field here because it's easier to deploy that way if we want a different owner.

Reply:
base branch is main here, it's already removed in develop/v2

Reply:
yes, rebasing right now